ophyd/areadetector/plugins.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, *args, **kwargs):
                                           self.plugin_type.get(), self.prefix))
 
         self.enable_on_stage()
+        self.stage_sigs.move_to_end('enable', last=False)
```

Comment:
Nice find on `move_to_end` (I'll pretend I had a reason not to use that when I came up with the hxntools solution above...)

Reply:
I didn't notice hxntools was a blue link or I would have used your function. To be honest I googled how to move an OrderedDict item to the beginning.